worlds/alttp/EntranceShuffle.py

```diff
@@ -1254,10 +1254,10 @@ def connect_reachable_exit(entrance, caves, doors):
     else:
         raise NotImplementedError('Shuffling not supported yet')
 
-    if world.glitches_required[player] in ['overworld_glitches', 'hybrid_major_glitches', 'no_logic']:
+    if world.worlds[player].options.glitches_required.value in ['overworld_glitches', 'hybrid_major_glitches', 'no_logic']:
```

Comment:
I don't think this works - The `.value` is gonna be an int, not a string.
I believe you can actually drop the `.value` here. Using `in` on a list or tuple just needs **equality** to be implemented. Only if it were a set instead of a list/tuple, there would be an issue.